### PR TITLE
Add frontend login setup

### PR DESCRIPTION
**Pull request by benjamin-ranft**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
   return (
-    <div>
-      Welcome
+    <UserContextProvider>
         <Switch>
             <Route path="/login" component={LoginPage}/>
+            <ProtectedRoute path="/discoveries" component={DiscoveriesPage}/>
             <Route path="/">
-                <Redirect to="/login"/>
+                <Redirect to="/discoveries"/>
```
Comment (FabianKeck) [inline frontend/src/App.js:15]: This does look strange to me, having to redirects in one Route

Comment (benjamin-ranft) [reply frontend/src/App.js:15]: @FabianKeck thanks for the feedback, but it's actually only <Redirect to="/discoveries"/> line 12, marked in red has been removed

**Comment by Asamya** [comment]
The code looks fine to me. I think a little bit of testing is in order :)

**Comment by benjamin-ranft** [comment]
I don't think we discussed any tests for the login frontend in class, or did we?

**Pull request merged**